Gaijinizer patch/CommonEvents_472.txt

```diff
@@ -4,157 +4,177 @@
 
 親友の仇に、気持ちよく中出しした気分はどうじゃ？
 > CONTEXT: CommonEvents_472/3/Dialogue/enemy17
-Hojo, you're right Buzama...
-How is the feeling of having a creampie
-comfortable for your close friend?#MTLed
+Hohoho, so unslightly...
+
+How does it feel to be pleasured by and ejaculate
+inside your foe?#Ron
 > END STRING
```

Comment:
"unslightly" -> "unsightly"

Reply:
Found another "unslightly" all the way back in Mukuro's dialogue, I'll fix that while I'm at it.